{"uuid": "37cacae0-ca81-452f-84e9-ceaf7f73dadb", "children": ["a296b3c2-03bd-49ba-af54-067e3b6dc418"], "befores": [{"name": "capture_screenshot_on_failure", "status": "passed", "start": 1741873414127, "stop": 1741873414127}], "afters": [{"name": "capture_screenshot_on_failure::0", "status": "broken", "statusDetails": {"message": "AttributeError: 'Function' object has no attribute 'rep_call'\n", "trace": "  File \"C:\\Users\\rutwik\\miniconda3\\envs\\automation\\lib\\site-packages\\allure_commons\\_allure.py\", line 221, in __call__\n    return self._fixture_function(*args, **kwargs)\n  File \"C:\\Users\\rutwik\\miniconda3\\envs\\automation\\lib\\site-packages\\_pytest\\fixtures.py\", line 907, in _teardown_yield_fixture\n    next(it)\n  File \"C:\\Users\\rutwik\\Automation\\conftest.py\", line 101, in capture_screenshot_on_failure\n    if request.node.rep_call.failed:\n"}, "start": 1741873414313, "stop": 1741873414316}], "start": 1741873414126, "stop": 1741873414316}